tutorial/README.md

```diff
@@ -206,6 +207,35 @@ docker-compose -f docker-compose-sqlserver.yaml exec sqlserver bash -c '/opt/mss
 docker-compose -f docker-compose-sqlserver.yaml down
 ```
 
+## Using DB2
+
+```shell
+# Start the topology as defined in http://debezium.io/docs/tutorial/
+export DEBEZIUM_VERSION=1.1
+
+# DEBEZIUM_DB2_VOLUME is an empty local directory;
+# Must be given as relative path, e.g. ./db2data
+export DEBEZIUM_DB2_VOLUME=<local persistent volume directory>
+
+docker-compose -f docker-compose-db2.yaml up
+
+# Start DB2 connector
+curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @register-db2.json
+
+# Consume messages from a Debezium topic
+docker-compose -f docker-compose-db2.yaml exec kafka /kafka/bin/kafka-console-consumer.sh \
+    --bootstrap-server kafka:9092 \
+    --from-beginning \
+    --property print.key=true \
+    --topic db2server.DB2INST1.CUSTOMERS
```

Comment:
DB2 is case sensitive. Standart all tables without quoted SCHEMA TABLE name will be automatically UPPERCASE. 
We have to be careful here because of the ASNCDC.IBMSNAP_REGISTER  entry and SQL UDF ADDTABLE / REMOVETABLE are absolutely case sensitive.

Reply:
Ok, thanks for clariying. What we have seems to make most sense then indeed.